server/container/guice/cassandra-guice/src/main/jib/glowroot/admin.json

```diff
@@ -0,0 +1,5 @@
+{
+  "web": {
+    "bindAddress": "0.0.0.0"
```

Comment:
Can't we share all glowroot conf in a single maven module?

Reply:
no the glowroot conf is not exactly the same (think of blob store not suitable for memory)